.travis.yml

```diff
@@ -2,7 +2,6 @@ language: python
 sudo: false
 python:
     - 2.7
-    - 3.4
```

Comment:
Additionally support for 3.8-dev could be added but this can be done in another PR.